src/common.js

```diff
@@ -7,6 +7,7 @@ var config = {
   silent: false,
   fatal: false,
   verbose: false,
+  noglob: false
```

Comment:
Suggestion (Totally optional): Maybe make this `glob`, and have it on by default?